doc/fireplace.txt

```diff
@@ -57,11 +57,19 @@ DOCUMENTATION                                   *fireplace-documentation*
 :Javadoc {class}        Open the java docs for the given class in a browser.
 
                                                 *fireplace-K*
-K                       Look up docs for symbol under cursor.
+K                       Look up (java) doc/spec-form for symbol/keyword under
+                        cursor.
 
                                                 *fireplace-:FindDoc*
 :FindDoc {arg}          Wrapper around (clojure.repl/find-doc ...).
 
+                                                *fireplace-:SpecForm*
+:SpecForm [kw]          Show form for spec at the cursor or given argument
+
+                                                *fireplace-:SpecExample*
+:SpecExample [kw]       Generate example for spec at the cursor or given
+                        argument
+
```

Comment:
`:SpecForm {keyword}`, and the wording needs tweaking but I'm not sure to what. Do these roughly correspond to `clojure.spec` functions? If so, it might make sense to reference that like we do for `:FindDoc`. I worry that just saying "spec" is confusing to those not familiar (it was for me).

Reply:
We could use some codified version like `(spec/form)` - they are at least all named the same for now in all three versions of spec, that are in the wild.

Reply:
I used `spec/form` and `gen/generate` - the later is the alias used in the main docs https://clojure.org/guides/spec#_sampling_generators; the first is a bit more clear than just `s`

Reply:
Can we go all in and use the full namespaces? That should make it easier to look up.

Reply:
The problem with the namespace would be to name all two currently (and soon three) supported by orchard/cider (spec "0", spec alpha, spec alpha 2) - which is a problem on it's on (see answer to "error handling" comment below)